DfE.FindInformationAcademiesTrusts/appsettings.json

```diff
@@ -14,8 +14,16 @@
     "ClientSecret": "",
     "TenantID": "",
     "RedirectUri": "",
+    "AllowedRoles": "fiat.create",
     "CallbackPath": "/signin-oidc"
   },
+
+  "TestOverride":
```

Comment:
Small change (if you happen to be touching this file again) - the start brackets in the rest of the json file are not on their own line, it would be good to make this new code consistent

Reply:
how odd it is on my file locally!

Reply:
it seems to have resolved its self with the latest commit